helpers

```diff
@@ -0,0 +1,15 @@
+// TODO: remove this file once Node 10 is EOL:
+const {
+  applyExtends,
+  cjsPlatformShim,
+  Parser,
+  processArgv,
+} = require('./build/index.cjs');
+
+module.exports = {
+  applyExtends: (config, cwd, mergeExtends) => {
+    return applyExtends(config, cwd, mergeExtends, cjsPlatformShim);
+  },
+  hideBin: processArgv.hideBin,
+  Parser,
+};
```

Comment:
i'm confused, why an extensionless file instead of a `helpers` dir with a `package.json` that has `"type": "commonjs"` and an `index.js` file?

The current approach will be way more likely cause problems with parsers and resolvers (admittedly, ones with bugs, but i can't even confidently say `resolve` would work properly with this)

Reply:
I appreciate that having a subfolder would cleanup the top level a bit too, we could pull the mjs in there